lib/ansible/modules/commands/script.py

```diff
@@ -48,6 +48,12 @@
     version_added: "2.4"
     required: false
     default: null
+  interpreter:
```

Comment:
To follow the conventions of `shell` and `command`, this should instead be `executable`

Reply:
Thanks for the review.  I've changed 'interpreter' to 'executable'.